src/Chapter3.hs

```diff
@@ -910,6 +1000,17 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = Gold Int
+
+instance Append Gold where
+    append (Gold a) (Gold b) = Gold $ a + b
+
+instance Append ([a]) where
+    append xs ys = xs ++ ys
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
    append = (++)
```